packages/block-editor/src/hooks/position.js

```diff
@@ -66,8 +66,29 @@ const FIXED_OPTION = {
 	),
 };
 
+const ZINDEX_OPTIONS = [
+	{
+		key: 'auto',
+		value: '',
+		name: __( 'Auto' ),
+		className: OPTION_CLASSNAME,
+	},
+	{
+		key: '5',
+		value: '5',
+		name: __( 'Top' ),
+		className: OPTION_CLASSNAME,
+	},
+	{
+		key: '-1',
+		value: '-1',
+		name: __( 'Bottom' ),
+		className: OPTION_CLASSNAME,
+	},
+];
+
 const POSITION_SIDES = [ 'top', 'right', 'bottom', 'left' ];
-const VALID_POSITION_TYPES = [ 'sticky', 'fixed' ];
+const VALID_POSITION_TYPES = [ 'relative', 'sticky', 'fixed' ];
```

Comment:
Another option is to not make "relative" a position value and just add `position: relative` whenever we add a z-index. 

Thinking out loud a bit here: adding relative position as a setting would only really be worth it if we were providing it as a way to scope an absolutely positioned block to its container, but given that some blocks (e.g.Cover) are already styled with position by default, it would be better for consistency's sake if we _always_ scope absolutely positioned blocks to their containers, in which case offering relative positioning becomes useles.

Reply:
Oh, good insights there! Agreed, I like the idea of implicitly using `relative` if no position type is set, rather than explicitly setting it 👍